content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "O3DE Visual Studio 2022 Support!"
+date: 2022-11-09
+slug: o3de-vs2022-initial-support
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+{{< image-width src="/images/blog/vs2022/vs2022-installer-v143-toolset.png" width="100%" alt="Visual Studio 2022 v143 toolset" >}}
+
+Because O3DE uses [CMake](/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
```

Comment:
change: `build using the Visual Studio 2022` -> `build using Visual Studio 2022`

Reply:
There is some irony here.

We can't figure out where it should be called "the O3DE" or "O3DE", but we are fairly certain it should be called "Visual Studio 2022" and not "Visual Studio 2022"

Is it the "The Batman" or just "Batman"?